app/build.gradle

```diff
@@ -0,0 +1,68 @@
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
+        classpath "org.sonarsource.scanner.gradle:sonarqube-gradle-plugin:${sonarqubePluginVersion}"
+    }
+}
+
+apply plugin: 'org.springframework.boot'
+apply plugin: 'io.spring.dependency-management'
+apply plugin: "org.sonarqube"
+apply plugin: 'application'
+
+mainClassName = "bonux.yada.YadaApplication"
+
+sourceCompatibility = 1.10
+targetCompatibility = 1.10
+
+group = 'bonux.yada'
+version = '0.0.1-SNAPSHOT'
```

Comment:
Imho snapshot pieliek klāt, kad ievieto jaru kādā nexusā. Jo snapshot var pārtapt par nesnapshot bez kommita.

Reply:
kad mums būs ci tad pie relīzēm snapšoti pārtaps par norm versijām. pagaidām snapšot it is :)